scripts/provisioning/s3.config.conf.tmpl

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Copyright

Reply:
I realized it later that we cannot add copyright to this file as this file is a json file, (confstore would load it as json file) and we cannot add comments to json file.

Reply:
I have changed format from  json to yaml of the template files as suggested, and added the copyright header as well.